src/wiki/forms.py

```diff
@@ -447,10 +447,9 @@ class PermissionsForm(PluginSettingsFormMixin, forms.ModelForm):
         help_text=_("Enter the username of the owner."),
     )
     group = forms.ModelChoiceField(
-        Group.objects.all(), empty_label=_("(none)"), label=_("Group"), required=False
+        Group.objects.all(), empty_label=_("(none)"), label=_("Group"), required=False,
+        widget=forms.Select(attrs={"class": "form-control"})
     )
-    if settings.USE_BOOTSTRAP_SELECT_WIDGET:
-        group.widget = SelectWidgetBootstrap()
```

Comment:
The idea here was to contain the boostrap-specific CSS etc in `SelectWidgetBootstrap`, so the customization should be contained there.

The django-wiki UI can run without Bootstrap this way:

1. Replace all templates
2. Set a configuration variable to disable custom form widgets

Reply:
After looking at the code and Bootstrap 4's dropdowns, I get it.. it's not nice to be so reliant on JavaScript to display something that the host platform handles much better.

I tried out Bootstrap 4's implementation, it's quite okay but some Keyboard shortcuts aren't nice. For instance you cannot jump to an item by its first letter.